### ISSUE TITLE
Reconstruct surface from Zernike moments 

### ISSUE THREAD

**Issue by fredericoschardong** [luispedro/mahotas#65]
It would be neat to have a function that reconstructs a surface based on the Zernike moments generated by mahotas.features.zernike_moments. In some cases one might want to compute the Euclidean distance between the original and the reconstructed image, thus justifying why this is useful.

Apparently someone has implemented such a thing (I have not tested):
https://github.com/tvwerkhoven/PyCourse/blob/master/python102/examples/py102-example2-zernike.py

**Comment by luispedro** [comment]
I agree it'd be very nice. ATM, I don't have the cycles to do it myself and it's not something I would need on any other project; so it's very unlikely I will do it.

The code you linked to unfortunately has the wrong license. Perhaps the author could be persuaded to relicense and adapt for mahotas, but, right now, it cannot be used :(

**Comment by fredericoschardong** [comment]
Too bad for me, I need this. To make things worse I am neither a python developer nor have any experience with Zernike polynomials.

**Comment by luispedro** [comment]
GPL2 is not compatible as mahotas is currently MIT-licensed, which does not have the copyleft restriction.

**Comment by fredericoschardong** [comment]
@luispedro There you go: http://stackoverflow.com/a/33339289/557697
